vllm/worker/worker.py

```diff
@@ -456,20 +457,22 @@ def get_cache_block_size_bytes(self) -> int:
 
 
 def init_worker_distributed_environment(
-    parallel_config: ParallelConfig,
+    config: VllmConfig,
```

Comment:
nit: rename to `vllm_config` for consistency with other code.

Reply:
Fixed. Thank you!